flow-typed/src/ChorusClient.js

```diff
@@ -0,0 +1,12 @@
+interface ChorusClient {
+	open(url: string): Promise<Event>,
+	close(): void,
+	connect(): void,
+	publishStep(
+		pattern: string,
+		callback: (...args: Array<string>) => number | string | void,
```

Comment:
In fact the return type could be more flexible but I think it's a very good idea to limit it to number, String or void at this point. There is an issue around how Chorus converts that to a Java number - I'll do some thinking on that but I'm likely to treat this as a BigDecimal I think